application/view/omeka/migrate/index.phtml

```diff
@@ -1,9 +1,13 @@
-<?php
-$translate = $this->plugin('translate');
-?>
-<h2><?php echo $translate('Update database'); ?></h2>
-<p><?php echo $translate('There was a recent software upgrade and the database needs to be updated. The
-entire site will be down for maintenance until you click the button below.'); ?></p>
+<h2><?php echo $this->translate('Update database'); ?></h2>
+<p><?php echo $this->translate('There was a recent software upgrade and the database needs to be updated. The entire site will be down for maintenance until you click the button below.'); ?></p>
 <form method="post">
-    <button name="submit"><?php echo $translate('Update database'); ?></button>
+    <button name="submit"<?php echo $environment->isCompatible() ? '' : ' disabled'; ?>><?php echo $this->translate('Update database'); ?></button>
```

Comment:
Maybe we just want to not show the form at all here if there's an error

again @kimisgold

Reply:
Note that the above paragraph says "The entire site will be down for maintenance _until you click the button below_." Iy would be confusing if there was no button.

Reply:
I'm fine with the disabled error button, especially in regards to Jim's point.